CHANGELOG.md

```diff
@@ -1,6 +1,16 @@
 Changelog
 =========
 
+## 1.4.1 (2024-02-28)
+
+### Bug fixes
+
+* Use ObjC strings instead of C strings to avoid ASAN lifetime race condition observed in React Native apps
```

Comment:
I don't think this was RN (as that doesn't use this native library)?

```suggestion
* Use ObjC strings instead of C strings to avoid ASAN lifetime race condition
```